test/solveEquation/solveEquation.test.js

```diff
@@ -114,17 +114,3 @@ describe('constant comparison support', function () {
   ];
   tests.forEach(t => testSolveConstantEquation(t[0], t[1], t[2]));
 });
-
-function testEquationError(equationString, debug=false) {
```

Comment:
maybe add another equation we can't solve, to make sure it's still throwing instead of doing weird stuff? or is there nothing left that falls into that category (where it tries to solve for a while and does weird stuff but doesn't actually solve)